doc/charm++/manual.rst

```diff
@@ -2894,96 +2922,84 @@ elements migratable (see migration section above) and choose a load
-   end of some iteration and when all elements reach that call load
-   balancing is triggered. The objects can start executing again when
-   ResumeFromSync() is called. In this case, the user redefines
-   ResumeFromSync() to trigger the next iteration of the application.
-   This manual way of using the at sync mode results in a barrier at
-   load balancing (see example here :numref:`lbexample`).
-
-#. **Manual mode**: The load balancer can be programmed to be started
-   manually. To switch to the manual mode, the application calls
-   *TurnManualLBOn()* on every processor to prevent the load balancer
-   from starting automatically. *TurnManualLBOn()* should be called as
-   early as possible in the program. It could be called at the
-   initialization part of the program, for example from a global
-   variable constructor, or in an initproc call
-   (Section :numref:`initproc`). It can also be called in the
-   constructor of a static array or before the *doneInserting* call for
-   a dynamic array. It can be called multiple times on one processor,
-   but only the last one takes effect.
-
-   The function call *CkStartLB()* starts load balancing immediately.
-   This call should be made at only one place on only one processor.
-   This function is not blocking, the object will continue to process
-   messages and the load balancing, when triggered, happens in the
-   background.
-
-   *TurnManualLBOff()* turns off manual load balancing and switches back
-   to the automatic load balancing mode.
+   until load balancing finishes. The user calls ``AtSync()`` at the
+   end of some iteration, then, when all elements reach that call,
+   load balancing is triggered. The objects can start executing again
+   when ``ResumeFromSync()`` is called. In this case, the user
+   redefines ``ResumeFromSync()`` to trigger the next iteration of the
+   application. This pattern effectively results in a barrier at load
+   balancing time (see example here :numref:`lbexample`).
+
+#. **Periodic load balancing mode**: This mode uses a timer to perform
+   load balancing periodically at a user-specified interval. In order
+   to use this mode, the user must provide the *+LBPeriod {period}*
+   runtime option, the *{period}* argument specifying the minimum time
+   between consecutive LB invocations in seconds. For example,
+   *+LBPeriod 10.5* can be used to invoke load balancing roughly every
+   10.5 seconds. Additionally, no array element can have
+   ``usesAtSync`` set to true. In this mode, array elements may be
+   asked to migrate at any time, provided that they are not in the
+   middle of executing an entry method. Thus, the PUP routines for
+   array elements must migrate all data needed to reconstruct the
+   object at any point in its lifecycle (as opposed to AtSync mode,
+   where PUP routines for load balancing migration are only called
+   after AtSync() and before ResumeFromSync(), so they can make some
```

Comment:
It sounds like periodic load balancing works with dynamic insertion. Could you very verbosely state whether or not that's true in the documentation?